### PR TITLE
Update commands in README.adoc

### LINKED ISSUE #583: Update commands in README.adoc
Update README.adoc to reflect updated shell command prefixes

### PR DESCRIPTION
**Pull request by chrisjs**
Fixes #583

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -77,7 +79,7 @@ The command `search` will search for all available packages. It should then show
 
 [source,bash,options="nowrap"]
 ----
-skipper:>search
+skipper:>package search
```
Comment (ilayaperumalg) [inline README.adoc]: We could also update the statement above as, The command `package search` will search...

Comment (chrisjs) [reply README.adoc]: good catch, thanks & updated!

**Comment by ilayaperumalg** [comment]
LGTM, merging.

**Comment by ilayaperumalg** [comment]
Squashed and merged as 0b1ec64

**Pull request closed without merging**